app/services/import_cait_indc.rb

```diff
@@ -59,36 +57,18 @@ def chart(indicator)
   def indicator_attributes(indicator)
     {
       chart: chart(indicator),
-      indicator_type: CaitIndc::IndicatorType.
-        find_by(name: indicator[:indicator_type]),
       category: CaitIndc::Category.find_by(name: indicator[:category]),
-      name: indicator[:indicator],
-      slug: Slug.create(indicator[:indicator]),
-      summary_list: indicator[:summary_list] == 'Yes',
-      on_map: indicator[:on_map] == 'Yes',
-      omit_from_detailed_view: indicator[:omit_from_detailed_view] == 'x',
-      show_in_dashboard: indicator[:show_in_dashboard] == 'x'
+      name: indicator[:long_name],
+      slug: indicator[:column_name],
+      on_map: @map.any? { |m| m[:indicator] == indicator[:column_name] }
```

Comment:
You probably should also store information about the Map category. See excel file *_Map.